pages/calculator.jsx

```diff
@@ -1,15 +1,43 @@
 import { Link } from "react-router-dom";
-
+import { useState } from "react";
 
 const Calculator = () => {
-    return(
-        <div className="count">
+  const [result, setResult] = useState("hii");
+
+  return (
+    <>
+      <Link to="/" className="start">
+        Startseite
+      </Link>
 
-<Link to="/" className="start">Home</Link>
+      <div className="container">
+        <form>
+          <input type="text" value={result} />
```

Comment:
This is not really done yet as far as I can tell, however, please add a `label` element 🙂